test/Npgsql.Tests/CommandTests.cs

```diff
@@ -996,7 +996,9 @@ public void Statements()
             // See also ReaderTests.Statements()
             using (var conn = OpenConnection())
             {
-                conn.ExecuteNonQuery("CREATE TEMP TABLE data (name TEXT) WITH OIDS");
+                var oidSupport = conn.PostgreSqlVersion.Major < 12;
+
+                conn.ExecuteNonQuery($"CREATE TEMP TABLE data (name TEXT){(oidSupport ? "WITH OIDS" : "")}");
```

Comment:
Per [the docs](https://www.postgresql.org/docs/current/sql-createtable.html):

> creating a table `WITH OIDS` is not supported anymore.

Reply:
This should be fixed by @Brar. See ##2713.

Reply:
Yeah, we can revert this. #2713 split this out into a dedicated test that only checks for OID, which is slightly better.

Reply:
FYI just merged #2713